RiotSharp.AspNetCore/Options/ApiKeyOptions.cs

```diff
@@ -1,14 +1,17 @@
-﻿using System;
+﻿using RiotSharp.AspNetCore.Caching;
+using System;
 using System.Collections.Generic;
 
-namespace RiotSharp.AspNetCore
+namespace RiotSharp.AspNetCore.Options
 {
     /// <summary>
     /// Specifies the options for ApiKey
     /// </summary>
     public class ApiKeyOptions
     {
-        internal ApiKeyOptions() { }
+        internal ApiKeyOptions() {
+            CacheType = CacheType.Internal;
```

Comment:
if internal is not recommended maybe we should use another as default?

Reply:
To be consistent with the rest of the lib, I'd put no cache by default.

Reply:
My bad. I wrongly thought that we use the internal implementation by default.